jib-core/build.gradle

```diff
@@ -51,6 +52,14 @@ dependencies {
 jar {
     manifest {
         attributes 'Implementation-Version': version
+        
+        // OSGi metadata
+        attributes 'Bundle-Name': 'Jib library for building Docker and OCI images'
+        attributes 'Bundle-Vendor': 'Google LLC'
+        attributes 'Bundle-SymbolicName': 'com.google.cloud.tools.jib'
+        attributes 'Bundle-DocURL': 'https://github.com/GoogleContainerTools/jib/wiki'
```

Comment:
Let's have at `https://github.com/GoogleContainerTools/jib` for now (the wiki is intended to be deprecated in favor of a better docs site later at https://googlecontainertools.github.io/jib).